logger.gd

```diff
@@ -224,7 +248,7 @@ const QUEUE_NONE = 0
 const QUEUE_ALL = 1
 const QUEUE_SMART = 2
 
-const FILE_BUFFER_SIZE = 30
+const FILE_BUFFER_SIZE = 30  # * TODO move ? @FILE
```

Comment:
This should likely be moved to `Logfile` yeah. And the `QUEUE_` constants should likely be moved to `ExternalSink` (likely as an enum since that should work well now in Godot).

Reply:
This is done in 50ed2bf